bin/pinto

```diff
@@ -17,6 +17,8 @@ use warnings;
 my $home_var = 'PINTO_HOME';
 my $home_dir = $ENV{$home_var};
 
+$ENV{GNUPGHOME} = $ENV{PINTO_GNUPGHOME} if $ENV{PINTO_GNUPGHOME};
```

Comment:
This seems out of place to me. `$ENV{GNUPGHOME}` should only be set when and where it is needed, and probably with `local` too.